lib/conjur/command/bootstrap.rb

```diff
@@ -28,111 +28,109 @@ class Conjur::Command::Bootstrap < Conjur::Command
 
   * Creation of a group called "security_admin".
 
-  * Giving the "security_admin" the power to manage public keys.
+	* Giving the "security_admin" the power to manage public keys.
 
-  * Creation of a user called "attic", which will be the owner of retired records.
+	* Creation of a user called "attic", which will be the owner of retired records.
 
-  * Storing the "attic" user's API key in a variable called "conjur/users/attic/api-key".
+	* Create system identities for use services such as pubkeys, rotator, and ldap-sync.
 
-  * (optional) Create a new user who will be made a member and admin of the "security_admin" group.
+	* (optional) Create a new user who will be made a member and admin of the "security_admin" group.
 
-  * (optional) If a new user was created, login as that user.
+	* (optional) If a new user was created, login as that user.
+
+  The Bootstrap command can be extended to perform additional actions by CLI plugins. The plugin just 
+  needs to define a new class in Conjur::Bootstrap::Command. Its "perform" method will be run automatically.
   )
   
-  # Determines whether the current logged-in user is sufficiently powerful to perform bootstrap.
-  # This is currently determined by detecting whether the logged-in role:
-  #
-  # * Is a user
-  # * Has admin privilege on the security_admin group role
-  # * Is an owner of the security_admin group resource
-  #
-  # The admin user will always satisfy these conditions, unless they are revoked for some reason.
-  # Other users created by the bootstrap command will (typically) also have these powers.
-  def self.security_admin_manager? api
-    return true if elevated?
-    
-    username = api.username
-    user = if username.index('/')
-      nil
-    else
-      api.user(username)
+  class BootstrapListener
+    def echo msg
+      $stderr.puts msg
     end
-    security_admin = api.group("security_admin")
-    
-    if user
-      memberships = user.role.memberships.map(&:roleid)
-      if security_admin.exists?
-        # The user has a role which is admin of the security_admin role
-        # The user has the role which owns the security_admin resource
-        has_admin?(user.role, security_admin.role) &&
-          memberships.member?(security_admin.resource.ownerid)
-      else
-        user.login == "admin"
-      end
-    else
-      false
+  end
+  
+  class << self
+    def quiet? options
+      !$stdin.tty? || options[:quiet]
     end
   end
 
   Conjur::CLI.command :bootstrap do |c|
     c.desc "Don't perform up-front checks to see if you are sufficiently privileged to run this command."
+    c.long_desc %Q(By default, 'bootstrap' switches to 'elevate' mode before it performs its commands.
```

Comment:
These two descriptions don't seem like they're describing the same functionality. Also, `--force` seems like a weird switch to use to say "run in a less privileged mode".